new-lamassu-admin/src/pages/Assets/Assets.js

```diff
@@ -0,0 +1,213 @@
+import Grid from '@material-ui/core/Grid'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableContainer from '@material-ui/core/TableContainer'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import { makeStyles, withStyles } from '@material-ui/core/styles'
+import * as R from 'ramda'
+import React from 'react'
+
+import TitleSection from 'src/components/layout/TitleSection'
+import { H4, Label2, P, Info2 } from 'src/components/typography'
+
+import styles from './Assets.styles'
+const useStyles = makeStyles(styles)
+
+const Cell = withStyles({
+  root: {
+    borderBottom: '4px solid white',
+    padding: 0,
+    paddingLeft: 20,
+    paddingRight: 20
+  }
+})(TableCell)
+
+const HeaderCell = withStyles({
+  root: {
+    borderBottom: '4px solid white',
+    padding: 0,
+    paddingLeft: 20,
+    paddingRight: 20,
+    backgroundColor: 'white'
+  }
+})(TableCell)
```

Comment:
You can move up the object containing
```
{
  borderBottom: '4px solid white',
  padding: 0,
  paddingLeft: 20,
  paddingRight: 20
}
```

And do something like this:
```suggestion
const Cell = withStyles({
  root: refactoredObj
})(TableCell)

const HeaderCell = withStyles({
  root: {
    ...refactoredObj
    backgroundColor: 'white'
  }
})(TableCell)
```